hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HAUtils.java

```diff
@@ -308,4 +311,15 @@ public static boolean isAccessControlException(Exception ex) {
     }
     return false;
   }
+
+  public static void checkSecurityAndSCMHAEnabled(OzoneConfiguration conf) {
```

Comment:
Can you tell me why secure SCM HA acceptance tests will fail?

As already we have a check for checking node list size > 1.

Reply:
I understand the goal is to prevent user from running into issues when before SCM HA+ security is fully supported? However, this also prevents us running any acceptance test with SCM HA + security in the mean while unless we wont add any here.

Reply:
Got it, introduced a new config which we can set in docker tests.